src/pages/match/MatchCandidatesTable.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import DataDisplay from '../../components/dataDisplays/DataDisplay';
+import { cellRendererTypes } from '../../components/dataDisplays/cellRenderers';
+
+const columns = [
+  {
+    // name: 'filename',
+    name: 'image_url',
+    labelId: 'FILENAME',
+  },
+  {
+    name: 'viewpoint',
+    labelId: 'VIEWPOINT',
+    align: 'left',
+  },
+  {
+    name: 'score',
+    labelId: 'SCORE',
+    align: 'left',
+    options: {
+      cellRenderer: cellRendererTypes.float,
+    },
+  },
+];
+
+export default function MatchCandidatesTable({
+  matchCandidates,
+  // selectedMatchCandidate,
+  setSelectedMatchCandidate,
+}) {
+  // console.log(matchCandidates);
+  return (
+    <DataDisplay
+      idKey="guid"
+      title="Match candidates"
+      data={matchCandidates}
+      // variant="secondary"
+      columns={columns}
+      noResultsTextId="NO_MATCH_CANDIDATES"
+      hideDownloadCsv
+      hideFilterColumns
+      onSelectRow={nextSelection => {
+        if (nextSelection) {
+          setSelectedMatchCandidate(nextSelection);
+        }
+      }}
```

Comment:
When is nextSelection falsy? Don't you want to remove the selected match candidate in that case?

Reply:
Well, right now if you click on a selected item in a table, it deselects that item. My goal here was to prevent it from deselecting the item, though it's still not working perfectly (I think I need to add an option make the table fully controlled)